webapp/src/js/controllers/analytics-target-aggregates-detail.js

```diff
@@ -0,0 +1,58 @@
+angular.module('inboxControllers').controller('AnalyticsTargetAggregatesDetailCtrl', function (
```

Comment:
I debated a lot on whether I should keep all this inside a single controller, so there wouldn't be a need for the additional Redux actions (though they were helpful during development, with the auto-logging). 
The reason I have two is mainly because that's how our whole app is structured. 
What do you think?